tests/subscripted_delays/test_subscripted_delays.mdl

```diff
@@ -100,7 +100,7 @@ delay[Apples]=
 		Simulation Control Parameters
 	|
 
-FINAL TIME  = 100
```

Comment:
Most of the tests can be run in one or two timesteps, as if the functionality works for 2 timesteps it will work for all of them (e.g. subscript aggregation). In delays, there is the possibility that an error could interact with how the simulation integration runs. 

I think we should probably leave this test to take the full time period, just to make sure we catch those issues. (running over longer time means that small errors are more likely to accumulate to a size we can detect).

Reply:
@JamesPHoughton  
I see, in any case this test was taking too long for me. Maybe we can reduce it more lets say to FINAL TIME=40.
I was trying to make the time step greater but 0.25 is the greater I was able to use, otherwise the model diverges...

Reply:
We can tweak some of the other values - give just two elements to each subscript, make delayN order 2,  etc. Does that give enough improvement? Making those changes wouldn't change the validity of the test, I don't think...

Reply:
I have the following problem:
when TIME is greater than 60 approx in Vensim's output we have small numbers of the order of 10⁻³ or 10⁻⁴ that when running in Python are 0. I think the error comes from a numerical error associated with the precision of operations in Python As when in DELAY N when I put the delay_time equal to the order the division is 1 and this problem dissapears. I know that there are some other problems with simple delays, maybe is the same.
Then I suggest making a shorter version of subscripted delays that checks the work of Delays with subscript without entering in ranges with numerical errors. And then make a second test mutch simple to study only the numerical error in DELAY N.